mne_bids/path.py

```diff
@@ -489,7 +548,8 @@ def _find_matching_sidecar(bids_fname, bids_root, suffix, allow_fail=False):
 
 def make_bids_basename(subject=None, session=None, task=None,
                        acquisition=None, run=None, processing=None,
-                       recording=None, space=None, prefix=None, suffix=None):
+                       recording=None, space=None,
+                       bids_root=None, suffix=None):
```

Comment:
we have broken backwards compatibility a little indiscriminately these days. Pinging @agramfort @hoechenberger @sappelhoff for opinions.